frontend/packages/core/src/dialog.tsx

```diff
@@ -1,27 +1,61 @@
-import React from "react";
-import MuiDialog from "@material-ui/core/Dialog";
-import DialogActions from "@material-ui/core/DialogActions";
-import DialogContent from "@material-ui/core/DialogContent";
-import DialogTitle from "@material-ui/core/DialogTitle";
-import Typography from "@material-ui/core/Typography";
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { DialogProps as MuiDialogProps } from "@material-ui/core";
+import {
+  Dialog as MuiDialog,
+  DialogActions as MuiDialogActions,
+  DialogContent as MuiDialogContent,
+  DialogTitle as MuiDialogTitle,
+  IconButton as MuiIconButton,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
 
-export interface DialogProps {
+const DialogTitle = styled(MuiDialogTitle)({
+  display: "flex",
+  justifyContent: "space-between",
+  fontSize: "16px",
+  padding: "16px 16px 0 16px",
+  fontWeight: 400,
```

Comment:
in Figma it's Roboto Medium 20px

```suggestion
  fontSize: "20px",
  fontWeight: 500,
  padding: "16px 16px 0 16px"
```